Jason/Jason-WorkShop-3/app/src/main/java/com/example/jason/jason_workshop_3/View/FeatureView/CheckBMIActivity.java

```diff
@@ -22,10 +25,12 @@
 public class CheckBMIActivity extends AppCompatActivity {
 
     private EditText edt_age, edt_weight, edt_height;
+    private TextView txv_hello;
```

Comment:
This naming is not java convention for android. it should be "tvHello". Same like above.